images/local-volume-node-cleanup/metadata.yaml

```
@@ -0,0 +1,12 @@
+name: local-volume-node-cleanup
+image: cgr.dev/chainguard/local-volume-node-cleanup
+logo: https://storage.googleapis.com/chainguard-academy/logos/local-volume-node-cleanup.svg
+endoflife: ""
+console_summary: ""
+short_description: The local volume node cleanup controller removes PersistentVolumes and PersistentVolumeClaims that reference deleted Nodes.
+compatibility_notes: ""
+readme_file: README.md
+upstream_url: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/4f81db77908ff67d8cac223c31413a293cd65d73/cmd/node-cleanup
```

```suggestion
upstream_url: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/cmd/node-cleanup
```

Since we'll be rebuilding latest it makes sense to link to the main branch so this is always current.